### PR TITLE
List maintainers and contributors

### PR DESCRIPTION
**Pull request by nicolaiparlog**
Went through the commits and created a list of contributions to address #188 and #226.

---

I hereby agree to the terms of the [JUnit Pioneer Contributor License Agreement](https://github.com/junit-pioneer/junit-pioneer/blob/master/CONTRIBUTING.md#junit-pioneer-contributor-license-agreement).

### CODE REVIEW

**Comment by nicolaiparlog** [comment]
Proposed commit message:

```
Many have been helping out with JUnitPioneer, some without ever
appearing in the commits and this project would not be where it is
(humble as that is) without their support. This commit lists all
current maintainers and past contributors and documents how to
continue that list.

Issue: #188 and #226
PR: #238
```

**Comment by nicolaiparlog** [comment]
Hey @mureinik, @beatngu13, @Hancho2009, @simonenkoi, @Bukama, @Michael1993, @aepfli, @panchenko, @sormuras, @beatngu13, @marcphilipp, @britter, @sormuras, and @marcphilipp: Thanks again for your contributions! :revolving_hearts: I listed them in the `README.md`. Let me know if you'd prefer not to be listed or want to change the description of your contribution.

In case this PR is already merged when you get around to replying, just open an issue and we will fix the text ASAP.

**Comment by nicolaiparlog** [comment]
@Bukama, @aepfli, @smoyer64: I need a paragraph for each of you. :)

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -40,8 +40,12 @@ Everybody uses a vocabulary and techniques that appear quite cryptic to those no
 We can't fix that in a short file like this, but we want to provide some pointers to get you started.
 If anything that follows in this document isn't clear, [open an issue](https://github.com/junit-pioneer/junit-pioneer/issues/new) and ask us to explain it better.
 
-First off, like for many open source projects, contributing code changes to JUnit Pioneer should be done via pull requests from a fork.
+* First off, like for many open source projects, contributing code changes to JUnit Pioneer should be done via pull requests from a fork.
 If you are not familiar with this concept, please have a look at the [GitHub help page](https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/working-with-forks).
+* `RAEDME.md` and `CONTRIBUTING.md` are written in Markdown.
```
Comment (Michael1993) [inline CONTRIBUTING.md:45]: Don't you mean README? :stuck_out_tongue:

File: README.md [hunk 2]
```diff
@@ -54,18 +70,54 @@ We welcome contributions of all shapes and forms! 🌞
...
+JUnit Pioneer, as small as it is, would be much smaller without kind souls contributing their time, energy, and skills.
+Thank you for your efforts! 🙏
+
+The least we can do is to thank them and list their accomplishments here (in lexicographic order).
+
+#### 2020
+
+* [Allon Murienik](https://github.com/mureinik) contributed [the range sources](https://junit-pioneer.org/docs/range-sources/) (#44 / #123)
+* [Daniel Kraus](https://github.com/beatngu13) contributed [the system property extension](https://junit-pioneer.org/docs/system-properties/) (#129 / #133)
+* [Hancho 2009](https://github.com/Hancho2009) controbuted the environment variable extension (#167 / #174)
```
Comment (Michael1993) [inline README.md:106]: ...controbuted or contributed? :thinking:

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -99,20 +103,39 @@ Yes, use it even if Jupiter's assertions are as good or better (c.f. `assertTrue
...
+Changes to project processes are usually reflected in these files (and are thus mostly done by maintainers).
+One aspect that's relevant to contributors is the list of contributions at the end of [README.md](README.md) - if you fixed a bug or added a feature, please add yourself to that list in the following form:
+
+```md
+* [$NAME]($GITHUB-URL) $CONTRIBUTIONS
+```
+
+* `$NAME` can be your actual name or your GitHub account name
+* `$GITHUB-URL` is the URL to your GitHub account
+* `$CONTRIBUTIONS` is a list of one or two contributions (possibly with an appended "and more" of there are more)
```
Comment (Bukama) [inline CONTRIBUTING.md:132]: > (possibly with an appended "and more" of there are more)

"if" there are more?

File: README.md [hunk 4]
```diff
@@ -54,18 +70,54 @@ We welcome contributions of all shapes and forms! 🌞
...
-Pioneer always depends on the lowest JUnit 5 version that supports its feature set, but that should not keep you from using 5's latest and greatest.
+<dl>
+	<dt>Matthias Bünger aka <a href="https://github.com/Bukama">Bukama</a></dt>
+	<dd>Maintainer since April 2020.</dd>
 
-For our own infrastructure, we rely on the following compile and test dependencies:
+	<dt>Nicolai Parlog aka  <a href="https://github.com/nicolaiparlog">nicolaiparlog</a></dt>
+	<dd>Java enthusiast with a passion for learning and sharing.
+		He does that in blog posts, articles, newsletters, and books; in <a href="https://twitter.com/nipafx">tweets</a>, repos, videos, and streams; at conferences and in-house trainings - more on all of that on <a href="https://codefx.org">codefx.org</a>.
+		That aside, he's best known for his haircut.
```
Comment (Bukama) [inline README.md:85]: and his pioneer hat! :D

**Comment by Bukama** [comment]
> @Bukama, @aepfli, @smoyer64: I need a paragraph for each of you. :)

```
<dd>(Always tries to become a better) Java developer, loves testing and reads <a href="https://twitter.com/bukamabish">tweets</a>. Became a maintainer in April 2020, after he "caused" (authored) too many <a href="https://github.com/junit-pioneer/junit-pioneer/issues">bishues</a>.</dd>
```

File: README.md [hunk 5]
```diff
@@ -11,6 +11,7 @@ A melting pot for all kinds of extensions to
 
 Check out [junit-pioneer.org](http://junit-pioneer.org), particularly [the documentation section](http://junit-pioneer.org/docs).
 
+
```
Comment (beatngu13) [inline README.md:14]: Nitpick: Is there a reason why some sections are separated with two linebreaks (like here) and others with one (like "Contributing")?

Comment (nicolaiparlog) [reply README.md:14]: There's always a reason, @beatngu13. :grin: To better navigate a file, I put two empty lines before each top-level section - in most documents, that's the level 2 headers.

**Comment by nicolaiparlog** [comment]
Thanks for spotting the typos, fixed 'em.

**Comment by aepfli** [comment]
> @Bukama, @aepfli, @smoyer64: I need a paragraph for each of you. :)

<dd>Bearded guy in Lederhosen, who loves to code, and loves to explore Code Quality, testing and other tools, which can improve the live of a software-craftsman. <a href="https://www.couchsurfing.com/people/simmens">Passionated couchsurfer</a> and <a href="https://www.facebook.com/togtrama">Hobby-Event-Planner</a>.</dd>

okay that is my take, not sure if couchsurfing and social club are appropriate links for a software project, but it humanizes me a little bit.

File: README.md [hunk 6]
```diff
@@ -54,18 +70,55 @@ We welcome contributions of all shapes and forms! 🌞
...
+JUnit Pioneer, as small as it is, would be much smaller without kind souls contributing their time, energy, and skills.
+Thank you for your efforts! 🙏
+
+The least we can do is to thank them and list their accomplishments here (in lexicographic order).
+
+#### 2020
+
+* [Allon Murienik](https://github.com/mureinik) contributed [the range sources](https://junit-pioneer.org/docs/range-sources/) (#44 / #123)
+* [Daniel Kraus](https://github.com/beatngu13) contributed [the system property extension](https://junit-pioneer.org/docs/system-properties/) (#129 / #133)
+* [Hancho 2009](https://github.com/Hancho2009) contributed the environment variable extension (#167 / #174)
```
Comment (Hancho2009) [inline README.md:107]:
```suggestion
* [Mark Rösler](https://github.com/Hancho2009) contributed the environment variable extension (#167 / #174)
```

**Pull request merged**